styles/styles.css

```diff
@@ -0,0 +1,132 @@
+
+.container{
+display: grid;
+grid-template: 1fr 2fr 2fr .5fr 2fr 2fr 3.5fr 2fr auto / 2fr 1fr 1fr;
+grid-template-areas: 
+"nav nav nav"
+"presentation presentation presentation"
+"quote1 quote1 quote1"
+"why_tittle why_tittle why_tittle"
+"why why why"
+"quote2 quote2 quote2"
+"team team team"
+"sponsor sponsor sponsor"
+"footer footer footer";
+}
+
+nav {
+    grid-area: nav;
+    margin: 20px;
+    flex-wrap: wrap;
```

Comment:
did you mean to put flex-wrap here?  usually flex-wrap lives in whatever ruleset display:flex lives.